theano/tensor/tests/test_opt.py

```diff
@@ -377,19 +377,19 @@ def test_elemwise_multiple_inputs_optimisation(self):
 #            (dz*fy*(fx+fy),(fx,fy,dz),(dxv,dyv,dzv),2,'float64'),#check mixed type mul
             # check with dimshuffle of constant
             (fx + fy + fz + 2, (fx, fy, fz), (fxv, fyv, fzv), 1, {'custom':
-                 'float32', 'numpy+floatX': config.floatX, 'numpy': 'float64'}),
+                 'float32', 'np+floatX': config.floatX, 'np': 'float64'}),
```

Comment:
Why do you revert all the fix you did in the previous commit? This is string, we don't want to change them to np. The issue was just about changing the code.